src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SyntaxTokenToSemanticTokensMethods.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright(c).NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Services
+{
+    internal static class SyntaxTokenToSemanticTokensMethods
```

Comment:
It's specific to the edits so I'd make the class specific about that. We don't want any other "helper" methods in this class anyhow given it's entire purpose is to isolate the logic that we grabbed from O#.